styles/variables-base.less

```diff
@@ -448,7 +448,7 @@
 @agate-incrementslider-horizontal-button-increment-border-width: 0;
 @agate-incrementslider-vertical-margin: @agate-slider-vertical-margin;
 @agate-incrementslider-vertical-slider-margin: 0;
-@agate-incrementslider-vertical-slider-padding: 0;
+@agate-incrementslider-vertical-slider-padding: (((@agate-slider-knob-width / 2) * @agate-slider-knob-resting-state-scale) + 3px) ((@agate-slider-height - @agate-slider-track-height) / 2);
```

Comment:
This file is _probably_ not the place for this amount of specificity. Its original home seems more appropriate, even if it needs to be completely duplicated into the gallium variables.